client/src/Switch.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import Login from "./LoginPage";
+import SignUp from "./SignUp";
+import './Switch.css';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import { BrowserRouter as Router, Route, Switch, Link } from 'react-router-dom';
+
+const Switcher = (props) => {
```

Comment:
I got a little confused that the file is called Switcher, but the component is called Switch. Can you make it consistent please?